lib/docs/filters/core/external_urls.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Docs
+  class ExternalUrlsFilter < Filter
+
+    def call
+      if context[:external_urls]
+
+        root = path_to_root
```

Comment:
Can we simply use the following?

```suggestion
        root = '/'
```